.github/config/evergreen-arm-softfp.json

```diff
@@ -12,23 +24,26 @@
       "platform":"evergreen-arm-softfp",
       "target_platform":"evergreen-arm-softfp",
       "target_cpu":"target_cpu=\\\"arm\\\"",
-      "extra_gn_arguments":"use_asan=false"
+      "extra_gn_arguments":"use_asan=false",
+      "bootloader_extra_gn_arguments": "sb_is_evergreen_compatible=true",
```

Comment:
This should be set in the platform's configuration.gni, not passed through gn args